### ISSUE TITLE
Using npx svelte-add@latest scss breaks vite

### ISSUE THREAD

**Issue by rpop0** [svelte-add/svelte-add#256]
Hello,

I started a new project with svelte kit and when configuring it I selected the TypeScript option. I then added scss using the svelte-add and everything stopped working, vite was just serving a 404 page.

After a bit of trial and error I noticed that under vite.config.ts i have a vite.config.js

![w5tZ1Jk 1](https://user-images.githubusercontent.com/38384209/222961768-9bdaff71-ad30-46c2-a297-012cfd4113c7.png)

Containing the following code:
```
const config = {
	css: {
		preprocessorOptions: {
			scss: {
				additionalData: '@use "src/variables.scss" as *;'
			}
		}
	}
};

export default config;
```

Deleting this file results in the app being served again.

**Comment by waffleflopper** [comment]
I was pulling out (metaphorically because I completely bald these days) my hair.  Was getting ready to give up.  Thanks for sharing this solution!  At first I didn't know why I was throwing the important analysis errors in the main App.svelte file and I was going down a rabbit hole.  Eventually figured out it happened after using svelte-add and ended up here.  I had a vite.config.js right under mine too that I didn't see.  

You're a lifesaver!

**Comment by manuel3108** [comment]
Duplicates #234

**Comment by manuel3108** [comment]
Will be fixed by #234 soon
